application/src/main/java/org/togetherjava/tjbot/commands/system/BotCore.java

```diff
@@ -47,13 +51,14 @@
  * event listener, using {@link net.dv8tion.jda.api.JDA#addEventListener(Object...)}. Afterwards,
  * the system is ready and will correctly forward events to all commands.
  */
-public final class BotCore extends ListenerAdapter implements SlashCommandProvider {
+public final class BotCore extends ListenerAdapter implements CommandProvider {
     private static final Logger logger = LoggerFactory.getLogger(BotCore.class);
     private static final String RELOAD_COMMAND = "reload";
     private static final ExecutorService COMMAND_SERVICE = Executors.newCachedThreadPool();
     private static final ScheduledExecutorService ROUTINE_SERVICE =
             Executors.newScheduledThreadPool(5);
     private final Config config;
+    private final List<UserInteractor> interactors;
```

Comment:
Please dont introduce this as separate list. instead role with `nameToInteractor.values()`. Better to have a single source of truth